scripts/st2bootstrap-el8.sh

```diff
@@ -492,6 +491,11 @@ install_st2_dependencies() {
   if [[ -z "$is_epel_installed" ]]; then
     sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
   fi
+
+  # Install rabbit from packagecloud
```

Comment:
Any background, eg why default Rabbitmq package didn't work? 
I think a better code comment would describe the reasons "why" rather than "what".